### PR TITLE
Fix spelling errors found in the documentation

### PR DESCRIPTION
**Pull request by jpreese**
Happy Tuesday, Boundary team! As I was going through the Boundary documentation, I noticed a few typos. Please let me know if you'd like anything else changed.

### CODE REVIEW

File: website/content/docs/getting-started/connect-to-target.mdx [hunk 1]
```diff
@@ -180,5 +180,5 @@ details.
 
 ## Next Steps
 
-See our [common workflows](/docs/common-workflows) for in depth discussion on managing scopes, targets,
+See our [common workflows](https://learn.hashicorp.com/collections/boundary/common-workflows) for in depth discussion on managing scopes, targets,
```
Comment (jpreese) [inline website/content/docs/getting-started/connect-to-target.mdx:183]: note: when clicking on this link it currently returns a `404`. Looking at the other link to `common-workflows` on the left side navbar, this seems the most appropriate.

**Comment by anoncam** [comment]
signed the CLA

**Comment by anoncam** [comment]
although i'm not sure why i was asked to. lol

**Comment by jefferai** [comment]
Thanks!

**Pull request merged**